### ISSUE TITLE
Deprecation of purestorage.fusion?

### ISSUE THREAD

**Issue by mariolenz** [Pure-Storage-Ansible/Fusion-Collection#167]
Greez from the Ansible community steering committee!

We've heard that [you have no plans for new purestorage.fusion releases and want to deprecate this collection](https://forum.ansible.com/t/3712).

If this is true, it would be really great if you could document this in your README, maybe even do a new / last release (1.16.1?) where you announce this and archive this repo.

**Comment by sdodsley** [comment]
Yes - we plan to deprecate this and will be updating the README shortly and issuing a new release based on that
